### DISCUSSION TITLE
Display value for relations [Jul 29]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6133] [Discord Threads]
> 🙋‍♂️ **thetruehero**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Display value for a relation can be configured using 'Set as Display Value', as detailed here   
#### https://docs.nocodb.com/setup-and-usages/display-value  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1123726372922200154)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
